src/modules/commands/discord/outfitscoreboard.ts

```diff
@@ -0,0 +1,307 @@
+import discord from 'discord.js'
+import { floor } from 'mathjs'
+import { OutfitAliasNotFoundError } from '@app/errors'
+import { constants } from '@app/global/constants'
+import { log } from '@app/utils/log'
+import { divide, mod } from '@app/utils/math'
+import { Command } from '@commands/CommandHandler'
+import { censusApi } from '@planetside/CensusApi'
+import { validateArgumentRange } from '../validators'
+
+export default new Command<discord.Message>({
+  keyword: 'outfitscoreboard',
+  description: 'show PS2 outfit player scoreboard',
+  help: "Usage:\n`{prefix}outfitscoreboard <alias> <stat>` - shows an outfit's player scoreboard\n`{prefix}outfitstats <alias> <stat> shame` - shows an outfit's players scoreboard starting from the worst",
+  callback: async ({ args, reply, raw, env }) => {
+    if (args.length === 0) {
+      return reply(env.command.getHelp(env.handler))
+    }
+    validateArgumentRange(args.length, 2, 3)
+
+    const removeReaction = async (reaction: discord.MessageReaction) => {
+      try {
+        await reaction.users.remove(raw.author)
+      } catch (error) {
+        log.error('Error removing reaction:', error)
+      }
+    }
+
+    const genScoreboardEmbed = (
+      outfitName: string,
+      displayArray: string[],
+      stat: string,
+      page: number,
+      len: number,
+    ): discord.MessageEmbed => {
+      const scoreboardEmbed = new discord.MessageEmbed()
+        .setColor('#647CC4')
+        .setTitle(
+          `**${outfitName}** - ${stat} ${
+            shame === 1 ? 'shame' : ''
+          } scoreboard  (Page N°${page + 1}/${len})`,
+        )
+        .addField(
+          stat,
+          displayArray.slice(0, displayArray.length / 2).join('\n'),
+          true,
+        )
+        .addField(
+          stat,
+          displayArray.slice(displayArray.length / 2).join('\n'),
+          true,
+        )
+        .setTimestamp()
+        .setFooter('Interactive')
+      return scoreboardEmbed
+    }
+
+    const outfitAlias = args[0].toLowerCase()
+
+    const linesPerDisplay = 15
+
+    const columns = 2
+
+    const timeout = 10 * 60 * 1000 // 10 minutes
+
+    const stat = args[1]
+      .replace(/\s+/g, ' ')
+      .split(' ')
+      .map((s) => s.charAt(0).toUpperCase() + s.slice(1).toLowerCase())
+      .join(' ')
+
+    const memberStats = await censusApi.getOutfitMembersStats(outfitAlias)
+    if (memberStats === null) throw new OutfitAliasNotFoundError()
+
+    memberStats.outfitMember = memberStats.outfitMember.filter(
+      (outfitMember) => {
+        return (
+          typeof outfitMember.character !== 'undefined' &&
+          typeof outfitMember.character.stats[8] !== 'undefined'
+        )
+      },
+    )
+
+    let scoreboard: typeof memberStats.outfitMember
+
+    const displayScoreboard: string[] = []
+
+    let shame = -1
```

Comment:
This little trick with `shame` being used for sorting arrays is neat, but I'd much prefer you just used `.reverse()` on the sorted stat array.